functions/src/routes/addonHost.ts

```diff
@@ -0,0 +1,22 @@
+import express from "express";
+// import getFirestore from "firebase-admin/firestore";
+// const db = getFirestore();
+// import Sse from "../services/serverSentEvents";
+// import RTDB from "../services/realtimeDB";
+
+const router = express.Router();
+
+router.get("/raffle", async (req, res) => {
+  res.sendFile("index.html", { root: "../functions" });
+});
+// router.get(
+//   "/raffle/listen",
+//   async (req: Request<unknown, unknown, unknown, { id: string }>, res) => {
+//     const { id } = req.query;
+//     // Sse.connect(id, res);
+//     // req.on("close", Sse.disconnect.bind(id));
+//     // RTDB.start(id);
+//   }
+// );
```

Comment:
zelfde als hierboven:
"weet niet precies wat al deze comments doen, maar of je moet het nog laten werken voordat het naar de alpha gaat, of ze moeten weg als je ze niet meer gebruikt"

Reply:
haal ze nu weg